samba/build/build_deb

```diff
@@ -7,6 +7,9 @@ if test "$DISTRO" != "debian" -a "$DISTRO" != "ubuntu"; then
 fi
 
 ## Get the desired CEPH_BRANCH/CEPH_SHA1 ceph repo
+# Filter out origin/
+CEPH_BRANCH=$(basename "$CEPH_BRANCH")
```

Comment:
we have a method for this in build_utils.sh https://github.com/ceph/ceph-build/blob/master/scripts/build_utils.sh#L8